src/porepy/fracs/fracture_network.py

```diff
@@ -0,0 +1,129 @@
+"""
+Module containing a wrapper function to generate fracture networks in 2d and 3d.
+"""
+from __future__ import annotations
+
+import warnings
+from typing import Optional, Union
+
+import porepy as pp
+from porepy.fracs.fracture_network_2d import FractureNetwork2d
+from porepy.fracs.fracture_network_3d import FractureNetwork3d
+from porepy.fracs.line_fracture import LineFracture
+from porepy.fracs.plane_fracture import PlaneFracture
+
+# Custom typings
+Fracture = Union[LineFracture, PlaneFracture]
```

Comment:
I'm not sure if that's going to work, since `Fracture` is the abstract class and `LineFracture` and `PlaneFracture` are the children classes. They are different types, although the inherit from the same class.